### PR TITLE
Enable rotation of environment texture

### PR DESCRIPTION
**Pull request by simonihmig**
See https://doc.babylonjs.com/how_to/use_hdr_environment#introduction (last example in paragraph)

### CODE REVIEW

File: addon/components/ecsy-babylon/scene.ts [hunk 1]
```diff
@@ -69,6 +76,15 @@ export default class EcsyBabylonScene extends DomlessGlimmerComponent<EcsyBabylo
     }
   }
 
+  rotateEnvironmentTexture(angle = 0) {
+    const texture = this.scene.environmentTexture;
+    texture.setReflectionTextureMatrix(
```
Comment (nickschot) [inline addon/components/ecsy-babylon/scene.ts]: tests say this might be null, apparently

Comment (simonihmig) [reply addon/components/ecsy-babylon/scene.ts]: Hm, yes. Why did my IDE not complain!? 🤔

Comment (simonihmig) [reply addon/components/ecsy-babylon/scene.ts]: Ah, sh*t. Did I mention that `any` really is the devil...

https://github.com/kaliber5/ember-ecsy-babylon/blob/master/addon/components/ecsy-babylon/scene.ts#L17

Comment (simonihmig) [reply addon/components/ecsy-babylon/scene.ts]: should be fixed now!

Comment (nickschot) [reply addon/components/ecsy-babylon/scene.ts]: `any` really is

**Comment by simonihmig** [comment]
> The floating dependency tests seem to fail because of typescript 3.8

Hm, you know what the issue is, or how to fix this?

**Comment by nickschot** [comment]
> > The floating dependency tests seem to fail because of typescript 3.8
> 
> Hm, you know what the issue is, or how to fix this?

It's actual problems in TS 3.8. Haven't been fixed yet.

**Pull request merged**